zenoh-ext/src/querying_subscriber.rs

```diff
@@ -179,8 +178,8 @@ impl<'a, 'b, Handler> QueryingSubscriberBuilder<'a, 'b, crate::UserSpace, Handle
     #[inline]
     pub fn query_selector<IntoSelector>(mut self, query_selector: IntoSelector) -> Self
     where
-        IntoSelector: TryInto<Selector<'b>>,
-        <IntoSelector as TryInto<Selector<'b>>>::Error: Into<Error>,
+        IntoSelector: TryInto<KeyExpr<'b>>,
```

Comment:
I'm not sure this will work out. A string like `myresource?_time=[..]` will not be valid anymore.

Reply:
Seems like an error after several renamings :-( Fixing

Reply:
Thank you!